README.md

```diff
@@ -69,3 +69,15 @@ $ forge --help
 $ anvil --help
 $ cast --help
 ```
+
+## Validate Upgrade Safety 
+1. Checkout to the deployed implementation commit 
+2. Copy the YieldDistributor to test/upgrades/<version>/YieldDistributor.sol
+3. Checkout to upgrade candidate version (A version that is strictly higher than the version in the previous step)
+4. Update the version in the options object of the script/upgrades/ValidateUpgrade.s.sol script
```

Comment:
Update the version in the options object of the "script/upgrades/ValidateUpgrade.s.sol" script

Reply:
addressed with 60104f9